client/components/PubPreview/PubPreview.tsx

```diff
@@ -159,7 +159,20 @@ const PubPreview = (props: Props) => {
 						</div>
 					)}
 
-					{showDescription && <div className="description">{pubData.description}</div>}
+					{showDescription && (
+						<div className="description">
+							{featureFlags.htmlPubHeaderValues ? (
+								<span
+									// eslint-disable-next-line react/no-danger
+									dangerouslySetInnerHTML={{
+										__html: pubData.htmlDescription ?? '',
```

Comment:
You'll probably want to do `pubData.htmlDescription ?? pubData.description` here, so that we fall back to the normal text description when the feature flag is set.